libs/sdk-ui-ext/src/internal/components/configurationPanels/LineChartBasedConfigurationPanel.tsx

```diff
@@ -60,4 +72,14 @@ export default class LineChartBasedConfigurationPanel extends BaseChartConfigura
             </BubbleHoverTrigger>
         );
     }
+
+    protected isDataPointsControlDisabled(): boolean {
+        const properties = this.props.properties?.controls;
+
+        return (
+            this.props.type === "combo" &&
+            properties?.primaryChartType === "column" &&
+            properties?.secondaryChartType === "column"
+        );
+    }
```

Comment:
this looks suspicious. why should the LineBasedConfigPanel check for type 'combo' and types of primary/secondary charts?

indeed, the test must be done somewhere but a more sane place is in `PluggableComboChart` in `renderConfigurationPanel`. you can have this test there and then pass the result down to reused lineBasedConfig panel as a prop (dataPointsControlDisabled)

Reply:
You're right but the ConfigurationPanelContent class hierarchy makes it ugly to create custom props for the individual config panel types. I'll try to work out a solution though.